### PR TITLE
Update: Purchase Page links + Add Content History menu link

### PR DESCRIPTION
**Pull request by Mcastres**
### What does it do?

- Replace Purchase page links.
- Add a Content History Menu Link for CE users only in order to display a purchase page.

### Why is it needed?

Give more visibility to our Enterprise features.

### How to test it?

- Start project in CE
You should see the Content History Menu link

- Start project in EE
You should not see the Content History Menu link.

### CODE REVIEW

**Review by HichamELBSI — APPROVED** [review]
Looks quite good, nice work @Mcastres!

File: packages/core/content-releases/admin/src/pages/PurchaseContentReleases.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const PurchaseContentReleases = () => {
               <LinkButton
                 variant="default"
                 endIcon={<ExternalLink />}
-                href="https://strapi.io/pricing-self-hosted?utm_campaign=Growth-Experiments&utm_source=In-Product&utm_medium=Releases"
+                href="https://strapi.io/enterprise?utm_campaign=In-Product-CTA&utm_source=Releases"
```
Comment (HichamELBSI) [inline packages/core/content-releases/admin/src/pages/PurchaseContentReleases.tsx:35]: Will we need to update these links when Growth plan will be there?

Comment (Mcastres) [reply packages/core/content-releases/admin/src/pages/PurchaseContentReleases.tsx:35]: No :)

**Review by simotae14 — APPROVED** [review]
It looks ok, well done 💪🏽

**Pull request merged**